prime-dotnet-webapi/Startup.cs

```diff
@@ -34,7 +35,7 @@
 using Prime.HttpClients.Mail;
 using Prime.Infrastructure;
 using Prime.ViewModels.HealthAuthorities;
-using Prime.ViewModels.HealthAuthoritySites;
+using Sentry.AspNetCore;

```

Comment:
The general grouping of `using` statements in the backend is all of the 3rd party imports and then the Prime namespaces; mind moving this `using` up? I'm not picky where it goes; I'm going to come by and alphabetize these at some point.